viktorsha/Frontend_Tasks
# V1-02. Local storage, notes editing

You can check out the site by following [this link](https://viktorsha.github.io/Frontend_Tasks/).
### **Overview** ###
There's a Datagrid with static data on the site. It contains 4 columns: _id_ of the note, _Title_, _Description_, and _Creation date_. 
One can select the row with specific data and check it out fully on the card placed on the right side of the page. 
The card contains full info about the note. Besides, one can click the "Edit note" button on the card, and the modal window with all information will be opened. Here one can update data and save it. 

### **Changes** ###
Added buttons "Edit note", which triggers edit function. Added modal window for editing notes. Added saving notes list to the local storage.

### **How to test** ###
1. Open the page by following [this link](https://viktorsha.github.io/Frontend_Tasks/)
2. Once the page is loaded a card on the right side should contain the text "Select note to display"
3. As soon as one selects the row, information from it should appear on the card.
4. To edit the note, click "Edit note"
5. In the opened modal window edit some data, click the "Save changes" button.
6. An updated data will be displayed on the card.
7. To check whether notes are stored in the local storage, open the developer's tools, click "Application" and "Local storage". There will be data under the "Notes" name with updated data.
